lib/tddium_client.rb

```diff
@@ -140,6 +149,8 @@ def call_api(method, api_path, params = {}, api_key = nil, retries = 5, xid=nil)
       tries = 0
       begin
         http = @client.send(method, tddium_uri(api_path), :body => call_params.to_json, :header => headers)
+      rescue OpenSSL::SSL::SSLError => e
```

Comment:
Thanks for notification!
Beside this: 
OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed

Also we might have there other thing, smth like: 
OpenSSL::SSL::SSLError: SSL_connect returned=1 errno=0 state=SSLv3 read server hello A: sslv3 alert handshake failure. 

In this case we will need to retry. I'll update code to handle self sign cert err only